Update torch flatbuffer usage to OSS version

qihqi opened the pull request:
Summary: Update users of flatbuffer serializer/loader to use the version in torch/csrc.

Test Plan:
sandcastle

Ran `buck run :test_models -- -k test_aten_relu` passes

Differential Revision: D33720611

suo commented:
Try rebasing on latest master to make the build errors go away.

gmagogsfm commented on torch/csrc/jit/mobile/flatbuffer_loader.h:
```diff
@@ -61,6 +61,12 @@ class FlatbufferLoader {
+      const std::string& type_str,
+      std::shared_ptr<CompilationUnit> cu);
+
+  void internal_registerTypeResolver(TypeResolver type_resolver);
```
If this is only meant for internal, could we consider making it private or protected?

qihqi replied:
it need to be called outside of FlatbufferLoader class.

gmagogsfm reviewed (commented):
I will take a look at internal diff too, which has a lot more changes.

qihqi committed:
Update torch flatbuffer usage to OSS version (#71957)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/71957

Update users of flatbuffer serializer/loader to use the version in torch/csrc.

Test Plan:
sandcastle

Ran `buck run :test_models -- -k test_aten_relu` passes

Reviewed By: gmagogsfm

Differential Revision: D33720611

fbshipit-source-id: 0c6f298f4ab1459be20595ae06d2e21036c66355
